src/Raven.Studio/wwwroot/Content/css/pages/admin-logs.less

```diff
@@ -4,8 +4,10 @@
             background-color: @panel-bg;
         }
         
-        &.custom-msg {
-            color: darken(@state-info-text, 7%);
+        span {
+            &.studio-item-info {
+                color: darken(@state-info-text, 7%);
```

Comment:
does it look good in other themes?

Reply:
Yes - I checked the other themes - it is a minor difference and it looks ok